packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -75,6 +77,21 @@ contract SortedOracles is ISortedOracles, ICeloVersionedContract, Ownable, Initi
     emit ReportExpirySet(_reportExpirySeconds);
   }
 
+  /**
+   * @notice Sets the report expiry parameter for a token.
+   * @param _token The address of the token to set
```

Comment:
Convention nit: period at the end.

Reply:
Also a wording nit: this sentence feels incomplete, add something like "expiry for" to the end? Or just shorten to "Address of the token".